pkg/setup/templates/1.13.go

```diff
@@ -0,0 +1,719 @@
+    schedule_delay = "0s"
+    startup_delay = "100ms"
+`),
+		},
+		{
+			Name:        "kubeconfig-insecure.yaml",
+			Destination: ManifestConfig,
+			Content: []byte(`---
+apiVersion: v1
+kind: Config
+clusters:
+  - cluster:
+      server: http://127.0.0.1:8080
+    name: p8s
+contexts:
+  - context:
+      cluster: p8s
+      user: p8s
+    name: p8s
+current-context: p8s
+users:
+  - name: p8s
+    username: p8s
+`),
+		},
+		{
+			Name:        "audit.yaml",
+			Destination: ManifestConfig,
+			Content: []byte(`---
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+rules:
+  - level: Request
+    verbs:
+      - create
+    omitStages:
+      - RequestReceived
+    resources:
+    - group: ""
+      resources:
+        - events
+
+  - level: Metadata
+    omitStages:
+      - RequestReceived
+`),
+		},
+		{
+			Name:        "admission.yaml",
+			Destination: ManifestConfig,
+			Content: []byte(`---
+kind: AdmissionConfiguration
+apiVersion: apiserver.k8s.io/v1alpha1
+plugins:
+- name: EventRateLimit
+  path: eventconfig.yaml
+`),
+		},
+		{
+			Name:        "eventconfig.yaml",
+			Destination: ManifestConfig,
+			Content: []byte(`---
+kind: Configuration
+apiVersion: eventratelimit.admission.k8s.io/v1alpha1
+limits:
+- type: Namespace
+  qps: 50
+  burst: 100
+  cacheSize: 2000
+- type: User
+  qps: 10
+  burst: 50
+`),
+		},
+		{
+			Name:        "kube-controller-manager.yaml",
+			Destination: ManifestAPI,
+			Content: []byte(`---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-controller-manager
+  namespace: kube-system
+automountServiceAccountToken: false
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: kube-controller-manager
+  name: kube-controller-manager
+  namespace: kube-system
+spec:
+  serviceAccountName: kube-controller-manager
+  automountServiceAccountToken: false
+  nodeName: "{{ .Hostname }}"
+  hostNetwork: true
+  volumes:
+  - name: secrets
+    hostPath:
+      path: "{{.RootABSPath}}/secrets"
+  containers:
+  - name: kube-controller-manager
+    image: "{{ .HyperkubeImageURL }}"
+    imagePullPolicy: IfNotPresent
+    command:
+    - /hyperkube
+    - controller-manager
+    - --master=http://127.0.0.1:8080
+    - --leader-elect=true
+    - --leader-elect-lease-duration=150s
+    - --leader-elect-renew-deadline=100s
+    - --leader-elect-retry-period=20s
+    - --cluster-signing-cert-file=/etc/secrets/pupernetes.certificate
+    - --cluster-signing-key-file=/etc/secrets/pupernetes.private_key
+    - --root-ca-file=/etc/secrets/pupernetes.issuing_ca
+    - --service-account-private-key-file=/etc/secrets/service-accounts.rsa
+    - --concurrent-deployment-syncs=2
+    - --concurrent-endpoint-syncs=2
+    - --concurrent-gc-syncs=5
+    - --concurrent-namespace-syncs=3
+    - --concurrent-replicaset-syncs=2
+    - --concurrent-resource-quota-syncs=2
+    - --concurrent-service-syncs=1
+    - --concurrent-serviceaccount-token-syncs=2
+    - --horizontal-pod-autoscaler-use-rest-clients=true
```

Comment:
This option is deprecated in 1.13 https://github.com/kubernetes/kubernetes/commit/8ef369ea2d1f487e46ca90f64dd30111eafafa78

Reply:
Thanks for the highlight.